src/main/webapp/i18n/de/student-dashboard.json

```diff
@@ -197,7 +197,13 @@
                 "complaintDue": "Beschwerde bis:",
                 "complaintDueTooltip": "Du kannst eine Beschwerde bis {{date}} einreichen.",
                 "complaintPossible": "Beschwerde möglich:",
-                "complaintPossibleTooltip": "Nach Erhalt einer Bewertung wirst du {{days}} Tage zum Einreichen einer Beschwerde haben."
+                "complaintPossibleTooltip": "Nach Erhalt einer Bewertung wirst du {{days}} Tage zum Einreichen einer Beschwerde haben.",
+                "chatWelcome": "Willkommen",
```

Comment:
I think it may make sense to create a separate file for these translation strings. 
Though at the very least you should indicate what is meant by "chat" here, by e.g. putting it into an "iris" subgroup

Reply:
Done